datamodel/src/main/java/com/hartwig/serve/datamodel/serialization/ServeJson.java

```diff
@@ -0,0 +1,45 @@
+package com.hartwig.serve.datamodel.serialization;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.datatype.guava.GuavaModule;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+import com.hartwig.serve.datamodel.RefGenome;
+import com.hartwig.serve.datamodel.ServeRecord;
+
+import org.jetbrains.annotations.NotNull;
+
+public final class ServeJson {
+    private static final ObjectMapper mapper = new ObjectMapper()
+            .registerModule(new GuavaModule())
+            .registerModule(new JavaTimeModule())
```

Comment:
Date's are serialized to json like this:

```
      "entryDate": [2016,1,19],
```

Which is the serializers default behaviour

Reply:
Why is this different from ACTIN itself where we use a lot of dates and also JSON serialization/deserialization? 
E.g. could we reuse things from `GsonSerializer` from ACTIN repo?

Reply:
We're moving away from GSON in general and prefer Jackson. The rationale is:
- In Java projects, Jackson has out of the box integration with @Immutables.
- Jackson provides more extensions and flexibility without having to define adapters. 
- Jackson is the library of choice for all web frameworks (jax-rs, resteasy, etc)